plugins/CoreHome/templates/_dataTableCell.twig

```diff
@@ -47,7 +47,7 @@
         {% if row.getMetadata('html_label_prefix') %}<span class='label-prefix'>{{ row.getMetadata('html_label_prefix') | raw }}&nbsp;</span>{% endif -%}
         {%- if row.getMetadata('html_label_suffix') %}<span class='label-suffix'>{{ row.getMetadata('html_label_suffix') | raw }}</span>{% endif -%}
 {% endif %}<span class="value">
-    {%- if row.getColumn(column) %}{% if column=='label' %}{{- row.getColumn(column)|rawSafeDecoded -}}{% else %}{{- row.getColumn(column)|number(2,0)|raw -}}{% endif %}
+    {%- if row.getColumn(column) or (column=='label' and row.getColumn(column) == "0") %}{% if column=='label' %}{{- row.getColumn(column)|rawSafeDecoded -}}{% else %}{{- row.getColumn(column)|number(2,0)|raw -}}{% endif %}
```

Comment:
> No it's not the same as the label column might still contain an empty string or null - which I guess should still be presented as -

I'm not sure how twig compares. When using two equal signs isn't `("0" == null) === true`? So currently `row.getColumn(column) or (column=='label' and row.getColumn(column) == "0")` should be same as `row.getColumn(column) or column=='label'` but not 100% sure. I don't think `same as` is needed for `label`. If a label can be an empty string or `null`, then we should compare via `is sameas('0')`